cmd/ipfs/daemon.go

```diff
@@ -677,6 +677,10 @@ func serveHTTPGateway(req *cmds.Request, cctx *oldcmds.Context) (<-chan error, e
 		opts = append(opts, corehttp.RedirectOption("", cfg.Gateway.RootRedirect))
 	}
 
+	if len(cfg.Gateway.PathPrefixes) > 0 {
+		log.Error("Support for X-Ipfs-Gateway-Prefix and Gateway.PathPrefixes is deprecated and will be removed in the next release: https://github.com/ipfs/go-ipfs/issues/7702")
```

Comment:
Maybe a note about "please comment on the issue if you're using this feature"? That way we can ask how and try to come up with an alternative/upgrade path without causing too much trouble. I don't think anyone actually _is_ using this feature, but still.